var/spack/repos/builtin/packages/meep/package.py

```diff
@@ -106,6 +112,11 @@ def configure_args(self):
         if "+single" in spec:
             config_args.append("--enable-single")
 
+        if "+openmp" in spec:
+            config_args.append("--with-openmp")
+        else:
+            config_args.append("--without-openmp")
+
```

Comment:
You can simplify this with one of the AutotoolsPackage helpers (see https://spack.readthedocs.io/en/latest/build_systems/autotoolspackage.html#with-or-without).

Reply:
Fixed && I also added a commit on top which simplifies two other trivial with/without options.